chesster.js

```diff
@@ -644,6 +647,78 @@ controller.hears([
     });
 });
 
+controller.hears([
+    'pairing ([a-zA-Z0-9]+) ([1-8])'
+], [
+    'direct_mention',
+    'direct_message'
+], function(bot, message) {
+    bot_exception_handler(bot, message, function() {
+        var player_name = message.match[1];
+        var round = message.match[2]
```

Comment:
This should default to the latest round so that people can just issue: pairing @lakinwecker and get their current round pairing